runusb

```diff
@@ -1,86 +1,28 @@
 #!/usr/bin/env python3
 
-import os.path
-import re
-import sys
-import select
 import logging
-import argparse
+from collections import namedtuple
+import select
+from enum import Enum
+import os
 import subprocess
-import collections
-
+import re
 
 LOGGER = logging.getLogger('runusb')
 
+PROC_FILE = '/proc/mounts'
+ROBOT_FILE = 'main.py'
+SPAWN_IMAGE = '/mnt/rootmirror'
+UPDATE_FILENAME = 'update.tar.xz'
+ROBOTD_LOCATION = '/var/robotd'
 
-def argument_parser():
-    parser = argparse.ArgumentParser(
-        description="Automatically run from mountpoints with .autorun",
-    )
-
-    parser.add_argument(
-        '-v',
-        '--verbose',
-        help="enable verbose output",
-        action='store_const',
-        const=logging.INFO,
-        dest='log_level',
-        default=logging.WARNING,
-    )
-
-    parser.add_argument(
-        '--debug',
-        help="enable very verbose output",
-        action='store_const',
-        const=logging.DEBUG,
-        dest='log_level',
-    )
-
-    parser.add_argument(
-        '--runfile',
-        help="name of the autorun file to detect",
-        action='store',
-        default='.autorun',
-    )
-
-    parser.add_argument(
-        '--spawn-image',
-        help="root filesystem for the container",
-        action='store',
-        default='/mnt/rootmirror',
-    )
-
-    parser.add_argument(
-        '--bind',
-        help="mountpoint(s) to bind inside the container",
-        action='append',
-        default=[],
-    )
-
-    parser.add_argument(
-        '--proc-mounts',
-        help="alternative path to watch from /proc/mounts",
-        action='store',
-        default='/proc/mounts',
-    )
-
-    parser.add_argument(
-        '--enable-network',
-        help="enable networking inside the container",
-        action='store_true',
-        dest='networking',
-    )
-
-    parser.add_argument(
-        '--disable-network',
-        help="disable networking inside the container (the default)",
-        action='store_false',
-        dest='networking',
-        default=False,
-    )
-
-    return parser
 
+Mountpoint = namedtuple('Mountpoint', (
```

Comment:
We only use the `mountpoint` attribute of this tuple. Is it worth having?

Reply:
I would think not, will factor out